HW1-Code.py

```diff
@@ -0,0 +1,98 @@
+import pandas as pd
+import numpy as np
+import plotly.express as px
+import os
+
+iris_df = pd.read_csv("D:/SDSU BDA/BDA 696/HW1/HW1-Data/iris.data.csv")
+iris_df.head
+
+#descriptive statistics
+iris_df.describe()
+##RESULTS: - has no columns              
+#             5.1         3.5         1.4         0.2
+# count  149.000000  149.000000  149.000000  149.000000
+# mean     5.848322    3.051007    3.774497    1.205369
+# std      0.828594    0.433499    1.759651    0.761292
+# min      4.300000    2.000000    1.000000    0.100000
+# 25%      5.100000    2.800000    1.600000    0.300000
+# 50%      5.800000    3.000000    4.400000    1.300000
+# 75%      6.400000    3.300000    5.100000    1.800000
+# max      7.900000    4.400000    6.900000    2.500000
+
+
+#Plots:
+#1)Scatter plot
+iris_df.columns = ['Sepal Length','Sepal Width', 'Petal Length', 'Petal Width', 'Class']
+iris_df.head
+
+
+fig = px.scatter(iris_df, x="Sepal Length", y="Sepal Width", color="Class", title= "Iris Test Plot")
+fig.show()
+
+fig0 = px.histogram(iris_df, x="Class", color="Class")
+fig0.show()
+
+
+
+#works but messy

```

Comment:
Why is this messy? Might be good to explain why its messy, even if it still works.

Reply:
Got it. Thanks. I am trying a trendline. It might go wrong. I will try this again tomorrow, but for now I am pushing some changes